p4-16/spec/P4-16-spec.mdk

```diff
@@ -2076,6 +2076,14 @@ enum bit<16> EtherType {
 introduces a new enumeration type, which contains five constants---e.g.,
 `EtherType.IPV4`.  This `enum` declaration specifies the fixed-width integer representation
 for each entry in the `enum` and provides an underlying type: `bit<16>`.
+This type of `enum` declaration can be thought of as declaring a new `bit<16>`
```

Comment:
Do we allow signed enum values as well?

Reply:
I've used unsigned in all of the examples, but I don't think there is any particular reason to restrict it to this.  Do you have an opinion?  (I'm happy to add a case for it.)